packages/vendor-core/package.json

```diff
@@ -25,7 +25,7 @@
     "scss"
   ],
   "dependencies": {
-    "@novnc/novnc": "^1.0.0",
+    "@novnc/novnc": "novnc/noVNC#25b3d49d322b0a7c9ee1e071d93042d70f5176b7 ",
```

Comment:
Once they will release your changes we should change it back to be consumed from npm?

If so, can you please add an item to the `roadmap.md` so we will have it documented?
https://github.com/theforeman/foreman-js/blob/master/roadmap.md#want-to-do-1

Information such as the targeted version and estimated time for the release would be useful.